### PR TITLE
[DS-1358] Add a new design for the Schedules block

### PR DESCRIPTION
**Pull request by aleevas**
Issue: [DS-1358](https://yusa.atlassian.net/browse/DS-1358)

### Steps for review:

- [ ] log as admin
- [ ] create a new page with LB
- [ ] add block Repeat (Group) Schedules
image

- [ ] fill all required fields
- [ ] save the layout
- [ ] verify that it looks according to new design

### CODE REVIEW

**Comment by rollki** [comment]
Related issue: https://www.drupal.org/project/openy_carnation/issues/3456336

**Review by rollki — APPROVED** [review]

File: modules/lb_repeat_schedules/package-lock.json
Comment (podarok) [inline modules/lb_repeat_schedules/package-lock.json:1]: I don't think it is a good idea to keep lock file in repo. Let's avoid it

File: openy_repeat.libraries.yml [hunk 1]
```diff
@@ -4,9 +4,9 @@ openy_repeat:
     js/repeat.js: {}
   css:
     base:
-      css/repeat.css: {}
-      css/print.css: { media: print }
-      css/repeat_mobile.css: {}
+      assets/css/repeat.css: {}
+      assets/css/print.css: { media: print }
+      assets/css/repeat_mobile.css: {}
```
Comment (podarok) [inline openy_repeat.libraries.yml:9]: todo: bump version on line 2 above for CDN invalidation

File: package-lock.json
Comment (podarok) [inline package-lock.json:1]: same. let's avoid lock files in repo, please

Comment (aleevas) [reply package-lock.json:1]: @podarok thanks, my bad. Removed

File: modules/lb_repeat_schedules/package-lock.json
Comment (aleevas) [reply modules/lb_repeat_schedules/package-lock.json:1]: @podarok  thanks, my bad. Removed

**Pull request merged**